meta-rdk-mtk-bpir4/recipes-ccsp/util/utopia.bbappend

```diff
@@ -3,14 +3,11 @@ include recipes-ccsp/ccsp/ccsp_common_bananapi.inc
 
 do_install_append() {
 
-    install -d ${D}${sysconfdir}/
-    install -d ${D}${sysconfdir}/utopia/	
-    DISTRO_WAN_ENABLED="${@bb.utils.contains('DISTRO_FEATURES','rdkb_wan_manager','true','false',d)}"
```

Comment:
do we need to remove these lines

Reply:
yes Chandra they are not required .i will create a fresh pul request closing this one